pinot-core/src/main/java/org/apache/pinot/core/transport/AsyncQueryResponse.java

```diff
@@ -181,4 +181,11 @@ void markServerDown(ServerRoutingInstance serverRoutingInstance, Exception excep
       markQueryFailed(serverRoutingInstance, exception);
     }
   }
+
+  /**
+   * Wait for one less server response
```

Comment:
NIT: update comment here. Wait for one less means "skip waiting for the response for the server"?